nightfall-administrator/src/index.mjs

```diff
@@ -20,8 +19,8 @@ async function main() {
   initUI();
   // start getting transaction information
   const signed = await start();
-  logger.info('******* Signed Transaction *******');
-  logger.info(signed);
+  console.log('******* Signed Transaction *******');
```

Comment:
Any reason you replaced our `logger` with `console.log` ?

Reply:
Yes, this is actually output that you always need (because you cut and paste it into an email etc to send to the person running the multisig).  If we try to use the logger to print it then that stops working once someone sets LOG_LEVEL>info.  Think of it as the programme's actual desired output (write to console), rather than information about how the programme is doing (log to logger).